### DISCUSSION TITLE
1.2.0

### DISCUSSION THREAD

**Discussion by Mattk70** [Mattk70/Chirpity-Electron#52] [Announcements]
- In-app notifications replace alerts
- Option for a system notification when lengthy analyses complete.
- Inserting a manual record now highlights the new record, and places the record in its position in the results list
- Removed options for BirdNET model which do not work with BirdNET (See #46 )

#### Bug fixes
- Fixed occasional issue where analyses did not complete. Fixes #49 
- Table header no longer shows when changing settings if no file is loaded.
- Charts fixed (oops!)
- You no longer need to click twice to retrieve saved results. Fixes #51 
- Analysing silence no longer throws an error when using the native chirpity model. Fixes #47 

<hr /><em>This discussion was created from the release <a href='https://github.com/Mattk70/Chirpity-Electron/releases/tag/v1.2.0'>1.2.0</a>.</em>

**Comment by Vollpflock** [comment]
beautiful :)

**Reply by Mattk70** [reply]
You’re welcome 😇
